DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,8 +56,13 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
-        // TODO: To be removed when major version is changed
+        // NEXT_MAJOR: remove this if block
         if (!isset($config['default_formatter'])) {
+            @trigger_error(<<<'EOT'
+Not setting the default_formatter configuration node is deprecated since 3.x,
```

Comment:
What would you do ? Concatenation? All on one line? I'll try concatenation if everyone is put off by this.

@sagikazarmark :

> Well, I don't really like this kind of line splitting within strings. Does not make things easier to read IMO.

By that, did you mean the string the code, or the produced string? I think I might have misunderstood this.